gibbon-compiler/src/Gibbon/L0/Typecheck.hs

```diff
@@ -68,7 +79,7 @@ tcProg prg@Prog{ddefs,fundefs,mainExp} = do
   pure prg { fundefs = fundefs'
            , mainExp = mainExp' }
 
-tcFun :: DDefs0 -> Gamma -> FunDef0 -> PassM FunDef0
+tcFun ::   DDefs0 -> Gamma -> FunDef0 -> PassM FunDef0
```

Comment:
There is a bunch of this `<space><space>` in the diff. How did it happen and can we get rid of them?